service/awsconfig/v1/version_bundles.go

```diff
@@ -0,0 +1,119 @@
+package v1
+
+import (
+	"time"
+
+	"github.com/giantswarm/versionbundle"
+)
+
+func VersionBundles() []versionbundle.Bundle {
+	return []versionbundle.Bundle{
+		{
+			Changelogs: []versionbundle.Changelog{
+				{
+					Component:   "calico",
+					Description: "Calico version updated.",
+					Kind:        versionbundle.KindChanged,
+				},
+				{
+					Component:   "docker",
+					Description: "Docker version updated.",
+					Kind:        versionbundle.KindChanged,
+				},
+				{
+					Component:   "etcd",
+					Description: "Etcd version updated.",
+					Kind:        versionbundle.KindChanged,
+				},
+				{
+					Component:   "kubedns",
+					Description: "KubeDNS version updated.",
+					Kind:        versionbundle.KindChanged,
+				},
+				{
+					Component:   "kubernetes",
+					Description: "Kubernetes version updated.",
+					Kind:        versionbundle.KindChanged,
+				},
+				{
+					Component:   "nginx-ingress-controller",
+					Description: "Nginx-ingress-controller version updated.",
+					Kind:        versionbundle.KindChanged,
+				},
+			},
+			Components: []versionbundle.Component{
+				{
+					Name:    "calico",
+					Version: "2.6.2",
+				},
+				{
+					Name:    "docker",
+					Version: "1.12.6",
+				},
+				{
+					Name:    "etcd",
+					Version: "3.2.7",
+				},
+				{
+					Name:    "kubedns",
+					Version: "1.14.5",
+				},
+				{
+					Name:    "kubernetes",
+					Version: "1.8.1",
+				},
+				{
+					Name:    "nginx-ingress-controller",
+					Version: "0.9.0",
+				},
+			},
+			Dependencies: []versionbundle.Dependency{},
+			Deprecated:   true,
+			Name:         "aws-operator",
+			Time:         time.Date(2017, time.November, 29, 16, 16, 0, 0, time.UTC),
+			Version:      "0.1.0",
```

Comment:
Please take your time for the review of this here and make sure you double checked the version bundle versions match the actual resource versions.

Reply:
I have a doubt here. For the legacy resource we also route empty version bundle versions to it. This is done in framework.go. Will this still work?

https://github.com/giantswarm/aws-operator/blob/aa1b3a96bd2474ba40c8907d12dc6c77d7bcd744/service/awsconfig/framework.go#L204-L206

Reply:
Right now HandleFuncs aren't removed.

Reply:
Ack. Thx for the confirmation.

Reply:
I will sort this in the next PR.